Tests/ObservationTests.swift

```diff
@@ -27,11 +27,12 @@ final class ObservationTests: XCTestCase {
         let expectation = self.expectation(description: #function)
         var publishedValue: String?
 
-        self.settings
-            .publisher(for: \.userId, options: [.new])
+        self.settings.$userId
             .sink { newValue in
                 publishedValue = newValue
-                expectation.fulfill()
+                if newValue == "test_publisher" {
+                    expectation.fulfill()
+                }
```

Comment:
Actually, could we leave this test as-is and write a new one, `test_Integration_projectedValue()` ?

I'd like to retain the `.publisher(...)` example in case folks prefer that